__tests__/auth-provider.test.ts

```
@@ -381,7 +381,7 @@ describe('AuthProvider', () => {
 			const tamperedData = Buffer.from('tampered_data').toString('base64');
 
 			// Tamper the Data
-			const tampered = serialized.replace(/:.*/g, `${tamperedData}:`);
+			const tampered = serialized!.replace(/:.*/g, `${tamperedData}:`);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)